src/main/java/org/zalando/nakadi/service/AuthorizationValidator.java

```diff
@@ -134,6 +134,27 @@ private void authorizeResourceAdmin(final Resource resource) throws AccessDenied
         }
     }
 
+    private void authorizeResourceView(final Resource resource) throws AccessDeniedException {
+        try {
+            if (!(adminService.isAdmin(AuthorizationService.Operation.ADMIN) ||
+                    authorizationService.isAuthorized(AuthorizationService.Operation.VIEW, resource))) {
+                throw new AccessDeniedException(AuthorizationService.Operation.VIEW, resource);
+            }
+        } catch (final PluginException e) {
+            throw new ServiceTemporarilyUnavailableException("Error calling authorization plugin", e);
+        }
+    }
+
+    public void authorizeEventTypeView(final EventType eventType)
```

Comment:
I don't understand why there are separate methods for event type view and subscription view.  IMO you can pass and authorize Resource directly since both these methods behave the same unless I am missing something.

Reply:
I follow the same pattern as other authorization functions in `AuthorizationValidator.java`, for example, `authorizeEventTypeWrite()`, `authorizeEventTypeAdmin()`, `authorizeSubscriptionAdmin()`. I think it is good as it does not mention `Resource` entity in places we call them.